tests.sh

```diff
@@ -71,10 +71,28 @@ function check_clean() {
     exit 1
 }
 
+function check_bad_build() {
+    echo "--- amethyst build"
+
+    cd mygame
+    rm -rf src
+    ../amethyst build
+
+    if [ $? -ne 0 ]; then
+        echo "--- Passed!"
+	echo
```

Comment:
There is something of a spacing issue here on line 83. Not very important, but my OCD just kicked in. :wink: